doc/changes/README.rst

```diff
@@ -0,0 +1,24 @@
+Recording Changes
+=================
+
+This directory contains "news fragments" which are small files containing text that will be integrated into release notes.
+The files can be in restructured text format or plain text.
+
+Each file should be named like ``<JIRA TICKET>.<TYPE>`` with a file extension defining the markup format.
+The ``<TYPE>`` should be one of:
+
+* ``feature``: New feature
+* ``bugfix``: A bug fix.
+* ``api``: An API change.
+* ``perf``: A performance enhancement.
+* ``doc``: A documentation improvement.
+* ``removal``: An API removal or deprecation.
+* ``other``: Other Changes and Additions of interest to general users.
```

Comment:
I think I decided that the distinction between other and misc was meaningless so I'm trying to standardize on misc.